### ISSUE TITLE
looping inclusion from one project to another?

### ISSUE THREAD

**Issue by calidion** [monero-project/monero#4541]
```
contrib/epee/include/net/abstract_tcp_server2.inl:54:96: fatal error: ../../../../src/cryptonote_core/cryptonote_core.h
```
what is the level epee belongs to?
`crypto` is based on `epee`.
while `epee` is based on `cryptonote_core` and `cryptonote_protocol` which are both based on `crypto` module.
it's quite confusing.

**Comment by moneromooo-monero** [comment]
Known, I'll get it fixed someday. core depends on epee.

**Comment by calidion** [comment]
@moneromooo-monero 

why it fails into some categories?

**Comment by moneromooo-monero** [comment]
Can you rephrase that ?

**Comment by calidion** [comment]
I mean these builds:

https://build.getmonero.org/builders/monero-linux-armv7/builds/1699

https://build.getmonero.org/builders/monero-linux-armv8/builds/813

**Comment by moneromooo-monero** [comment]
Click on "stdio" in the fiurst link, gets you to https://build.getmonero.org/builders/monero-linux-armv7/builds/1699/steps/compile/logs/stdio
At the end of the file, it shows:
g++: error: missing argument to ‘-march=’

**Comment by calidion** [comment]
how to fixed this problem?

**Comment by moneromooo-monero** [comment]
For this one, apply PR 4565.

**Comment by calidion** [comment]
Does this mean after pr 4565 is being merged, these errors will disappear?

**Comment by moneromooo-monero** [comment]
+resolved
